pkg/codegen/hcl2/rewrite_apply_test.go

```diff
@@ -16,6 +16,7 @@ func (nameInfo) Format(name string) string {
 	return name
 }
 
+//nolint: lll
```

Comment:
Nit:

```suggestion
// nolint: lll
```

Reply:
The lack of a space is proper style for machine-readable comments in Go. See https://github.com/golangci/golangci-lint#nolint for canonical examples of this particular comment--I'm trying to dig up the general rule, but haven't yet.

Reply:
> Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.

Reply:
Ah, we have a bunch of uses with the space that we should cleanup, then.